pkg/varlinkapi/system.go

```diff
@@ -90,22 +88,25 @@ func (i *LibpodAPI) GetInfo(call iopodman.VarlinkCall) error {
 		Graph_status:         graphStatus,
 	}
 
-	if len(info) > 2 {
-		registriesInterface := info[2].Data["registries"]
-		if registriesInterface != nil {
-			registries = registriesInterface.([]string)
-		}
-	}
-	if len(info) > 3 {
-		insecureRegistriesInterface := info[3].Data["registries"]
-		if insecureRegistriesInterface != nil {
-			insecureRegistries = insecureRegistriesInterface.([]string)
+	if len(info) > 1 {
+		for key, val := range info[2].Data {
+			if key == "search" {
+				podmanInfo.Registries.Search = val.([]string)
+				continue
+			}
+			fmt.Println("DAN4", key, val)
```

Comment:
I think that's a debugging/dev artifact and should be removed.

Reply:
removed